src/server/pachyderm_test.go

```diff
@@ -1209,6 +1209,30 @@ func TestPipelineFailure(t *testing.T) {
 	require.True(t, strings.Contains(jobInfo.Reason, "datum"))
 }
 
+func TestPPSEgressURLOnly(t *testing.T) {
+	if testing.Short() {
+		t.Skipf("Skipping %s in short mode", t.Name())
+	}
+	t.Parallel()
+	c, _ := minikubetestenv.AcquireCluster(t)
+
+	repo := tu.UniqueString(t.Name())
+	require.NoError(t, c.CreateRepo(repo))
+	require.NoError(t, c.PutFile(client.NewCommit(repo, "master", ""), "file1", strings.NewReader("foo")))
+
+	pipeline := tu.UniqueString("egress")
+	pipelineReq := basicPipelineReq(pipeline, repo)
+	pipelineReq.Egress = &pps.Egress{URL: fmt.Sprintf("test-minio://%s/%s/%s", minikubetestenv.MinioEndpoint, minikubetestenv.MinioBucket, pipeline)}
+
+	_, err := c.PpsAPIClient.CreatePipeline(c.Ctx(), pipelineReq)
+	require.NoError(t, err)
+	commitInfo, err := c.WaitCommit(pipeline, "master", "")
```

Comment:
It would be good to wrap this in a `NoErrorWithin`, so that it times out sooner if something breaks